bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/agreements/AgreementAcceptanceService.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2021 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.api.agreements;
+
+import java.util.function.Supplier;
+
+public interface AgreementAcceptanceService {
```

Comment:
since ?

Reply:
The package is declared as `internal` and does not demand version. Do we indeed need it here?

Reply:
Ah! I missed that